plugins/modules/sns_topic.py

```diff
@@ -287,9 +295,17 @@ def _topic_arn_lookup(self):
                 return topic
 
     def _create_topic(self):
+        attributes = { 'FifoTopic': 'false' }
```

Comment:
A bunch of other attributes could go in here, but are alrady taken care of in the update methods further down that also to validity checking etc., so I left it at that for a create.